lib/benchee/configuration.ex

```diff
@@ -122,6 +123,11 @@ defmodule Benchee.Configuration do
       equivalent to the behaviour Benchee had pre 0.5.0)
     * `:before_scenario`/`after_scenario`/`before_each`/`after_each` - read up on them in the hooks section in the README
     * `:measure_function_call_overhead` - Measure how long an empty function call takes and deduct this from each measure run time. Defaults to true.
+    * `profile_after` - accepts any of the following options:
+      * a boolean   - `true` will enable profiling with the default profiler
+      (`:cprof`) and `false` will disable profiling. Defaults to `false`.
+      * a profiler  - either as a tuple of `{profiler, opts}` (e.g., `{:fprof, [sort: :own]}`)
+      or just the profiler, which is equivalent to `{profiler, []}`.
```

Comment:
thanks for thinking about adding it here :+1: I'm still somewhat sad about that duplication between README and here :cry: